docs/docs/services/qix-engine/access-control.md

```diff
@@ -31,16 +50,86 @@ docker run
     -S SystemDenyRulePath=/<container folder>/deny.txt
 ```
 
-## Allow versus Deny
+## User
 
-In addition to using _Allow_ rules to grant access, the Qlik Associative Engine
-also supports _Deny_ rules to deny access.
+All rules are evaluated in the context of a _User_. In the rules syntax, the user is represented by the `user`
+object.
 
-!!! Note
-    _Deny_ rules are always applied first, if none matched
-    it will continue with _Allow_ rules.
+### User Attributes
+
+`user` contains all attributes defined in the
+[JWT header used to authenticate the user](../../tutorials/authorization.md#json-web-token).
+
+##### Examples
+
+```json
+{
+  "sub": "john-doe",
+  "name": "John Doe",
+  "employeeType": "developer",
+  "tags": ["research"],
+  "custom": {
+    "country": "sweden"
+  }
+}
+```
+
+Here, `employeeType`, and `custom.country` become attributes on `user` and can be used in a rule condition like this:
+
+```c
+user.employeeType = "developer" and user.custom.country = "sweden"
+```
+
+With the example JWT above, this rule evaluates to `true`.
+
+### User Built-in Functions
+
+#### `IsAnonymous()`
+
+Boolean function for user conditions that returns `true` only if the user requesting access is logged in as anonymous.
```

Comment:
I wonder what this means in the Core context. The JWT defines a `sub`, would that value be `anonymous` for this to evaluate to `true`?